community-membership.md

```diff
@@ -296,5 +297,72 @@ with the project. When a contributor returns to being more active, they may be
 promoted back to their previous position at the discretion of the current
 maintainers following the process defined above.
 
+## Specification Sponsor
+
+Specification sponsors are trusted collaborators of the technical committee, and
+work to review, approve, and sponsor [opentelemetry-specification][] issues and
+PRs.
+
+Members are defined
+in [Community Members -> Specification and Protos](./community-members.md#specifications-and-proto)
+and are members of the `spec-sponsors` team (TODO: add link to github team when
+available). Specification sponsors SHOULD list
+their [areas of interest](./areas-of-interest.md).
+
+### Requirements
+
+Specification sponsors are trusted to drive specification work which can be
+complex and highly impactful to downstream language SIGs and to the
+OpenTelemetry project as a whole. Approvals from specification sponsors count
+towards the approvals required to merge specification PRs. Therefore, there is a
+high bar to becoming a specification sponsor.
+
+- Contributor to OpenTelemetry project for at least 1 year
+- Has been the reviewer or author (through the sponsorship of another
+  specification sponsor) of at least 10 substantial PRs to specification or
+  related repositories
+- Demonstrated sound technical judgement, and a deep understanding of the
+  specification and the OpenTelemetry project
+- Nominated by technical committee member (
+  see [becoming a specification sponsor](#becoming-a-specification-sponsor))
+
+### Responsibilities and privileges
+
+- Review, respond to, and approve issues / PRs when tagged or when within areas
+  of interest.
+- Sponsor issues, taking responsibility to be point on requirement gathering,
+  developing solutions and prototypes, writing PRs, and responding to /
+  incorporating PR feedback. This can be done by the sponsor themselves, or by
+  another collaborator who is sponsored. If sponsoring another collaborator, the
+  sponsor should work with them to ensure the work is consistent with existing
+  specification.
+- Demonstrate sound technical judgement.
+
```

Comment:
```suggestion
- Frequently attend the specification SIG meeting where issues and PRs are synchronously discussed.

```
This might be implicit but I think it would be worth pointing this out as a responsibility.